samples/meetings-share-to-stage-signing/csharp/Source/MeetingSigning.Web/Models/DocumentDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+namespace Microsoft.Teams.Samples.MeetingSigning.Web.Models
+{
+    using System;
+    using Microsoft.Teams.Samples.MeetingSigning.Domain.Models;
+
+    public class DocumentDTO
```

Comment:
Question, why is this a distinct DTO from the DocumentListDTO? Why not just represent this as a DocumentListDTO with only one element?